haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/dao/SpecterDao.java

```diff
@@ -1,4 +1,65 @@
 package cz.muni.fi.pa165.hauntedhouses.dao;
 
+import cz.muni.fi.pa165.hauntedhouses.model.Ability;
+import cz.muni.fi.pa165.hauntedhouses.model.House;
+import cz.muni.fi.pa165.hauntedhouses.model.Specter;
+
+import java.util.List;
+
+/**
+ * @author Zoltan Fridrich
+ */
 public interface SpecterDao {
+
+    /**
+     * Creates new specter in the database.
+     * @param specter New specter.
+     */
```

Comment:
Dohodli sme sa, že javadoc (nie len pre túto funkciu) bude obsahovať aj informácie o tom, keď sa niečo vo funkcii pokazí (napr zavoláš create na entitu, ktorá už je v DB). Inšpiruj sa našimi implementáciami